py4DSTEM/io/native/read/read_v0_12.py

```diff
@@ -93,7 +93,7 @@ def read_v0_12(fp, **kwargs):
         # Parse optional arguments
         if 'mem' in kwargs.keys():
             mem = kwargs['mem']
-            assert(mem in ('RAM','MEMMAP'))
+            assert(mem in ('RAM','MEMMAP', 'DASK'))
```

Comment:
also edit the docstring - line 39, the 'mem' argument - to reflect the 'DASK' option

Reply:
Updated the doc string